lib/sprockets/commoner/processor.rb

```diff
@@ -47,6 +47,15 @@ def self.call(input)
         instance(input[:environment]).call(input)
       end
 
+      def self.configure(env, *args, **kwargs)
+        env.postprocessors['application/javascript'].each do |processor|
```

Comment:
Just like I said in my issue I created, I am not sure if `env.postprocessors['application/javascript']` will always be defined so this may raise an error if it tries to call `each` on nil. It might be worth it to throw in a nil check here